src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -556,12 +621,27 @@ void DlgPrefSound::masterLatencyChanged(double latency) {
     update();
 }
 
-void DlgPrefSound::headDelayChanged(double value) {
+void DlgPrefSound::masterDelaySpinboxChanged(double value) {
+    m_pMasterDelay->set(value);
+}
+
+void DlgPrefSound::headDelaySpinboxChanged(double value) {
     m_pHeadDelay->set(value);
 }
 
-void DlgPrefSound::masterDelayChanged(double value) {
-    m_pMasterDelay->set(value);
+void DlgPrefSound::boothDelaySpinboxChanged(double value) {
+    m_pBoothDelay->set(value);
+}
+
+void DlgPrefSound::roundTripLatencySpinboxChanged(double value) {
+    m_pRoundTripLatency->set(value);
+    roundTripLatencyChanged(value);
+}
+
+void DlgPrefSound::roundTripLatencyChanged(double newRoundTripLatency) {
+    double inputLatencyCompensation = newRoundTripLatency / 2.0;
```

Comment:
It is possible that there is a very slight difference in the latency for AD and DA conversion, but I think that would generally be on the order of a few samples. AFAIK there is no way for a user to measure this, so the best assumption is that the input latency is half the round trip latency measured with a loopback cable.